src/xenia/emulator.h

```diff
@@ -116,6 +116,12 @@ class Emulator {
   // Terminates the currently running title.
   X_STATUS TerminateTitle();
 
+  // takes array of emulated device paths, local paths, and symlinked drives
+  // handles initializing emulated devices
+  X_STATUS SetupVFSDevice(std::wstring devicepath, std::wstring localpath,
```

Comment:
Arguments not on the same column, also these should take u16string views, not wstrings.